polyglot/piranha/src/lib.rs

```diff
@@ -89,7 +90,7 @@ pub fn execute_piranha(
 
   let source_code_units = flag_cleaner.get_updated_files();
 
-  if should_rewrite_files {
+  if *configuration.dry_run() {
```

Comment:
```suggestion
  if !*configuration.dry_run() {
```
We want to persist files only when it is **not** a dry run, right?

Reply:
not `should_rewrite_files` and `dry_run` have reverse semantics.